avm/res/app/container-app/main.bicep

```diff
@@ -194,32 +191,6 @@ resource containerApp 'Microsoft.App/containerApps@2023-05-01' = {
   }
 }
 
-resource containerApp_diagnosticSettings 'Microsoft.Insights/diagnosticSettings@2021-05-01-preview' = [for (diagnosticSetting, index) in (diagnosticSettings ?? []): {
```

Comment:
Hey @oZakari, how come you removed the diagnostic settings? Are they actually not supported on the resource?

Reply:
If they are, please add it again - and also make sure it's part of the `max` & `waf-aligned` tests.

Reply:
They are not supported at the container app unfortunately, only at the managed/app environment. See here for reference: https://github.com/microsoft/azure-container-apps/issues/382